extensions/adobe/experience/adcloud/campaign.schema.json

```diff
@@ -18,6 +18,11 @@
           "type": "string",
           "description": "Identifier for the account."
         },
+        "xdm:dspId": {
```

Comment:
not thrilled with an acronym being all lower case...

Reply:
The "id" of all concepts should be capitalized. In this case, it should be dspID. Make changes to these references everywhere in the PR.

Reply:
I was afraid to be inconsistent with the other ids already in the schema. Wouldn't that be a problem?